.yarn/versions/b14843cf.yml

```diff
@@ -0,0 +1,4 @@
+releases:
+  "@data-wrangling-components/core": major
```

Comment:
I would actually release this as "minor", since you are adding new API functionality but not breaking backwards-compatibility